### PR TITLE
add skeleton Tagger page

### LINKED ISSUE #287: add a skeleton Tagger page
Add a skeleton page for the Tagger.

### PR DESCRIPTION
**Pull request by dwhieb**
## Related Issue

closes #287

## Description

This PR adds a skeleton Tagger page with no content.

This branch was made from the [issue-268 branch](https://github.com/digitallinguistics/app/tree/issue-268). PR #284 should be reviewed and merged first.

### CODE REVIEW

File: src/app/Nav/Nav.test.js [hunk 1]
```diff
@@ -3,27 +3,34 @@ describe(`Nav`, () => {
   it(`changes pages`, () => {
```
Comment (vadekh) [inline src/app/Nav/Nav.test.js:3]: Now that we have a number of pages, do we want to check to make sure they are all present in the Nav?

Comment (dwhieb) [reply src/app/Nav/Nav.test.js:3]: Yes, good call! Opened #293 for this.

**Comment by dwhieb** [comment]
@vadekh Updated from `master`, no other changes made. All tests passing locally and on CI. Skipping additional review.

**Comment by dwhieb** [comment]
[[app] add skeleton Tagger page](https://trello.com/c/oCgl8hH6/339-app-add-skeleton-tagger-page)

**Pull request merged**